recipes/nclpy/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "nclpy" %}
+{% set version = "0.0.9" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 0a19d99f2df42f99e2eba2382c90a20f89080408cb7d06d465378171cabdd678
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
```

Comment:
Looks like the package also imports `setuptools` in `setup.py`, so can you add that under `host`?